core/diirt/diirt-features/pom.xml

```diff
@@ -30,7 +30,6 @@
         <!-- Plugin required to properly publish source feature -->
         <groupId>org.eclipse.tycho</groupId>
         <artifactId>tycho-p2-plugin</artifactId>
-        <version>${tycho.version}</version>
```

Comment:
Hello Nick,
I've removed it because my Eclipse IDE shows a warning: "Overriding managed version 0.23.1 for tycho-p2 plugin". I don't like warnings, I usually consider them like errors to be fixed, and if we don't trust the IDE I wonder why are we still using it.
You're right, all other xx-features/pom.xml have it, and this is one of the reason my IDE is telling we have 5732 warnings. According the IDE, all these `<version>${tycho.version}</version>` should be removed.
If I'm wrong, please tell me and I'll put the missing line back.